### ISSUE TITLE
`lastEdit...` parameters haven't changed for deleted user

### ISSUE THREAD

**Issue by Antreesy** [nextcloud/spreed#11401]
Although the lastEdit parameters haven't changed for deleted user, but that's on a backend side:
![image](https://github.com/nextcloud/spreed/assets/93392545/d26ace7e-f196-4fb0-8522-fb05b42247d0)

_Originally posted by @Antreesy in https://github.com/nextcloud/spreed/pull/11361#discussion_r1457499895_

**Comment by nickvergessen** [comment]
yeah I would have noticed while working on the last missing point of the API ticket, handling the display name of editors, as deleting a user is exactly the problem :D
